src/components/PlioQuestion.vue

```diff
@@ -131,28 +149,65 @@ export default {
     
     // Highlights the correct option as green, wrong one as red
     showResult(){
-      if (!this.isAnswerCorrect) 
-        this.$refs[this.selectedOption].className = "answer_option-wrongAnswer"
+      if (!this.isAnswerCorrect) {
+        this.$refs[this.selectedOption].classList.add("wrongAnswer")
+      }
+      
+      this.$refs[this.correctAnswer].classList.add("correctAnswer")
+    },
+
+    removeOptionHighlight(){
+      var allOptions = document.querySelectorAll('.answer_option')
+
+      allOptions.forEach( (option) => {
+        option.removeAttribute("class");
+        option.className = "answer_option"
+      });
+    },
+
```

Comment:
user returns to a previously answered question ->
sees that his old response is highlighted ->
chooses another option and clicks submit ->
**new answers need to be highlighted**

For the last step, we need to remove existing highlighting, if there is any